modules/apps/object/object-test/src/testFunctional/macros/ObjectAdmin.macro

```diff
@@ -37,6 +37,18 @@ definition {
 			locator1 = "Button#TAB_SUBMIT_BTN");
 	}
 
+	macro addLayoutTabViaUI {
```

Comment:
Hey, @Italo-gabriel this macro already exists, can you search by:  _ObjectAdmin.addTabFieldsOnLayout_

Reply:
Done !